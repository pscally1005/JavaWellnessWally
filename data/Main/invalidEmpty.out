Enter a number to select: -------------------
NUTRITION LOG
-------------------
BMI CALCULATOR
-------------------
PACE CALCULATOR
-------------------
SPLIT CALCULATOR
-------------------
NUTRITION FACTS GENERATOR
-------------------
