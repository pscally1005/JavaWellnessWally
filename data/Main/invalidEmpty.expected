Enter a number to select: 